packages/chain-adapters/src/ChainAdapterManager.ts

```diff
@@ -90,6 +90,22 @@ export class ChainAdapterManager {
     return Array.from(this.supported.keys())
   }
 
+  getSupportedAccountTypes(): Record<ChainTypes, Array<UtxoAccountType | undefined>> {
```

Comment:
this should call the individual adapters and return what they respond with, otherwise this is just coupling the adapters to the manager?